src/hb-bit-page.hh

```diff
@@ -179,13 +199,18 @@ struct hb_bit_page_t
   typedef unsigned long long elt_t;
   static constexpr unsigned PAGE_BITS = 512;
   static_assert ((PAGE_BITS & ((PAGE_BITS) - 1)) == 0, "");
+  static constexpr unsigned PAGE_BITS_LG_2 = 9;
```

Comment:
Just use `LOG` instead of `LG`. Also more readable if you add this and and the other one in a separate commit.